video/external/src/decoder/openh264.rs

```diff
@@ -1,13 +1,62 @@
 use core::slice;
 use std::ffi::{c_int, c_uchar};
+use std::fmt::Display;
 use std::ptr;
+use std::sync::Arc;
 
 use crate::decoder::openh264_sys::{self, videoFormatI420, ISVCDecoder, OpenH264};
 use crate::decoder::VideoDecoder;
 
 use ruffle_render::bitmap::BitmapFormat;
 use ruffle_video::error::Error;
 use ruffle_video::frame::{DecodedFrame, EncodedFrame, FrameDependency};
+use thiserror::Error;
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct H264Version(u32, u32, u32);
```

Comment:
I feel like `OpenH264Version` would be a better name, lest it be confused with https://en.wikipedia.org/wiki/Advanced_Video_Coding#Versions .

Reply:
Although, there's already `openh264_sys::OpenH264Version`. I know it's not `Display` but it`s at least `Eq`...

Reply:
I don't think I should use `openh264_sys::OpenH264Version` as `openh264_sys` is private, I changed the name to `OpenH264Version`

Reply:
It could still be `pub use`'d, no? Anyway, if you decided not to use it, that's okay, I just wanted to make sure it's a deliberate thing. Thanks for the rename!